### ISSUE TITLE
Which fork of this tool is actually up to date?

### ISSUE THREAD

**Issue by GloriousDuck** [doxxx/ffxiv-craft-opt-web#426]
Do any of them have added standard touch as a combo action yet? Having a hard time finding the most up to date one.

**Comment by CoupDeKai** [comment]
If you go to Insights tab and then click on Network it'll bring up a graph  that is sorted to show the main repository at the top and then the forks are sorted by most recently updated. You'll have to drag sideways on the graph to see the most up-to-date stuff. The dots on the lines represent updates, so you can see that even though ramtur01 and riftwebdev are more recent they've done basically nothing as far as changes are concerned. The fork from echoeida looks promising though. It mentions combo actions in the changelogs.

A bit more digging shows echoeida listing is itself a fork of a fork of a fork, and that this website likely hosts a better working version of the tool: [https://notranged.github.io/#/simulator](https://notranged.github.io/#/simulator)

**Comment by GloriousDuck** [comment]
Yeah I've been using the notranged one lately but it bugs out sometimes still. It does seem to have the new comboactions included so that's good. Thanks for the Insights thing, didn't know about that.

**Comment by NotRanged** [comment]
I'll be trying to update it to endwalker soon:tm:

**Comment by NotRanged** [comment]
It's been updated for endwalker. The new actions are still missing and I'll add them soon, but the solver works again and- other than a slight rounding error in quality (e.g. 7982 instead of 7954 quality on a craft) it's all functional.

It's been an interesting journey to get it all working, had to write a new Python API scraper from scratch (something I've never done before), and I'm still rather new to javascript. But hey, it all works again!

https://notranged.github.io/#/simulator / https://github.com/NotRanged/NotRanged.github.io

e: new abilities added too

**Comment by EmaldIsBack** [comment]
@NotRanged you're awesome man, thanks!

**Comment by OppieTV** [comment]
Hello,

Since the update, I regularly fail in my crafts; the site does not take into account the passage of interior calm from an action to a line, same for the name of the elements, etc.

How can I get the solver to work properly again? I'm building the kitchen (60+) and I can't get any craft right, it's frustrating

**Comment by NotRanged** [comment]
Don't use this tool but go to https://notranged.github.io/#/simulator and use that one instead.

**Comment by OppieTV** [comment]
Thanks for the updated link but is it normal not to have access to the recipe library? https://i.imgur.com/29vYo4Y.png

Edit : My bad i see, it's only work in english; i will try to translate but thx for that anyway ! :)
